.gitignore

```diff
@@ -52,6 +52,8 @@ autoconf/autom4te.cache
 /CMakeSettings.json
 # CLion project configuration
 /.idea
+# Ignore .time files generated by polybench testing
+polygeist/tools/cgeist/Test/polybench/**/*.time
```

Comment:
This looks like a bug in the tests to me.
All artifacts produced by the tests must be emitted inside `build` directory (line 30 of this file already have it "ignored").

Reply:
I modified the tests to remove those *time temp. files.